### PR TITLE
Add a GitOps example for deploying the Redpanda Helm Chart with Flux

### PR DESCRIPTION
**Pull request by JakeSCahill**
Fixes https://github.com/redpanda-data/documentation-private/issues/1647

We will pull this into the docs site after https://github.com/redpanda-data/documentation-private/issues/1993 is completed.

### CODE REVIEW

**Review by pfcurtis — APPROVED** [review]

File: kubernetes/gitops/README.adoc [hunk 1]
```diff
@@ -0,0 +1,142 @@
+= Set Up GitOps for the Redpanda Helm Chart
+:description: In this example, you will use Flux to deploy the Redpanda Helm chart on a local Kubernetes cluster.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
:description: This example uses Flux to deploy the Redpanda Helm chart on a local Kubernetes cluster.
```

File: kubernetes/gitops/README.adoc [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+:page-interactive-example: https://play.instruqt.com/manage/redpanda/tracks/redpanda-k8s-gitops
+:page-category: Deployment, GitOps
+
+GitOps is a modern approach to managing and automating the deployment and provisioning process using Git as the single source of truth. It involves storing configuration files and deployment scripts in a Git repository, and then using automation tools to continuously monitor the repository for changes.
+
+In this example, you will use Flux to deploy the Redpanda Helm chart on a local Kubernetes cluster. Flux is a toolkit for GitOps with Kubernetes clusters that supports:
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
This example uses Flux to deploy the Redpanda Helm chart on a local Kubernetes cluster. Flux is a toolkit for GitOps with Kubernetes clusters that supports the following:
```

File: kubernetes/gitops/README.adoc [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+- *Version control for configurations*: Allowing you to track changes, collaborate, and revert to previous configurations if needed.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
- *Version control for configurations*: You can track changes, collaborate, and revert to previous configurations if needed.
```

File: kubernetes/gitops/README.adoc [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+- *Drift detection and remediation*: Continuously monitoring the Redpanda cluster's state. If discrepancies are detected, Flux can automatically remediate them to bring the Redpanda cluster back to the desired state.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
- *Drift detection and remediation*: Flux continuously monitors the Redpanda cluster's state. If discrepancies are detected, Flux automatically remediates them to bring the Redpanda cluster back to the desired state.
```

File: kubernetes/gitops/README.adoc [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+helm version
+----
+
+- https://kind.sigs.k8s.io/docs/user/quick-start/#installation[kind]
+
+- https://docs.docker.com/get-docker/[Docker]
+
+== Create a local Kubernetes cluster
+
+In this step, you create one master and three worker nodes (one worker node for each Redpanda broker).
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
Create one master and three worker nodes (one worker node for each Redpanda broker).
```

File: kubernetes/gitops/README.adoc [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+. Create the Kubernetes cluster from the configuration file:
++
+```bash
+kind create cluster --config kind.yaml
+```
+
+== Run the example
+
+To run the example, you must fork the `redpanda-examples` repository and configure Flux to connect to your fork.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
Fork the `redpanda-examples` repository, and configure Flux to connect to your fork.
```

File: kubernetes/gitops/README.adoc [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+The bootstrap script does the following:
+
+. Creates a deploy token and saves it as a Kubernetes Secret.
+. Creates an empty GitHub project, if the project specified by `--repository` doesn't exist.
+. Generates Flux definition files for your project.
+. Commits the definition files to the specified branch.
+. Applies the definition files to your cluster.
+. Applies the manifests in `kubernetes/gitops` which deploy Redpanda and cert-manager.
+
+After you run the script, Flux will be ready to manage itself and any other resources you add to the GitHub project at the specified path.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
After you run the script, Flux is ready to manage itself and any other resources you add to the GitHub project at the specified path.
```

File: kubernetes/gitops/README.adoc [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+Replace `<github-username>` with your GitHub username.
+
+The bootstrap script does the following:
+
+. Creates a deploy token and saves it as a Kubernetes Secret.
+. Creates an empty GitHub project, if the project specified by `--repository` doesn't exist.
+. Generates Flux definition files for your project.
+. Commits the definition files to the specified branch.
+. Applies the definition files to your cluster.
+. Applies the manifests in `kubernetes/gitops` which deploy Redpanda and cert-manager.
```
Comment (micheleRP) [inline kubernetes/gitops/README.adoc]:
```suggestion
. Creates a deploy token and saves it as a Kubernetes Secret
. Creates an empty GitHub project, if the project specified by `--repository` doesn't exist
. Generates Flux definition files for your project
. Commits the definition files to the specified branch
. Applies the definition files to your cluster
. Applies the manifests in `kubernetes/gitops` which deploy Redpanda and cert-manager
```

File: README.md [hunk 9]
```diff
@@ -2,7 +2,7 @@
 
 # Redpanda Examples
 
-A collection of examples to demonstrate how to interact with Redpanda from various clients and languages. Redpanda is Kafka API compatible. Any client that works with Kafka will work, but we have tested the ones listed [here](https://docs.redpanda.com/docs/reference/faq/#what-clients-do-you-recommend-to-use-with-redpanda).
+A collection of examples to demonstrate how to use Redpanda to accomplish goals using various clients, languages, and deployment environments. Redpanda is Kafka API compatible. Redpanda is compatible with Apache Kafka versions 0.11 and later, with specific exceptions noted on [this page](https://docs.redpanda.com/current/develop/kafka-clients/).
```
Comment (micheleRP) [inline README.md]:
```suggestion
A collection of examples to demonstrate how to use Redpanda with various clients, languages, and deployment environments. Redpanda is compatible with Apache Kafka versions 0.11 and later, with specific exceptions noted on [this page](https://docs.redpanda.com/current/develop/kafka-clients/).
```

**Review by micheleRP — APPROVED** [review]
I know this was really for the tooling, but I added some text suggestions since it was open!

**Pull request closed without merging**